test_complete_system.py

```
@@ -0,0 +1,252 @@
+        print(f"❌ 세션 관리: FAIL - {test_results['session_management']['details']}")
+    
+    # Redis 연결 확인
+    try:
+        import redis
+        r = redis.Redis(host='localhost', port=6379, db=0)
+        if r.ping():
+            score += 1
+            print("✅ Redis 연결: PASS")
+        else:
+            print("❌ Redis 연결: FAIL")
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)